### PR TITLE
e2e-test: add e2e tests and CRDs for ApisixClusterConfig v2

### PR DESCRIPTION
**Pull request by lingsamuel**
Signed-off-by: Ling Samuel <lingsamuelgrace@gmail.com>

- [x] New feature provided

### CODE REVIEW

**Review by tao12345666333 — APPROVED** [review]
LGTM

File: test/e2e/scaffold/cluster_config.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	cc := fmt.Sprintf(_apisixClusterConfigTemplate, s.opts.APISIXClusterConfigVersion, name, enable)
+	if err := k8s.KubectlApplyFromStringE(s.t, s.kubectlOptions, cc); err != nil {
+		return err
+	}
+	time.Sleep(5 * time.Second)
+	return nil
+}
+
+// DeleteApisixClusterConfig removes an ApisixClusterConfig CRD
+func (s *Scaffold) DeleteApisixClusterConfig(name string, enable bool) error {
```
[the remark this answers is not in the document]
Comment (lingsamuel) [reply test/e2e/scaffold/cluster_config.go:51]: I don't think it's necessary, this two functions are tiny enough

**Comment by tao12345666333** [comment]
ping @gxthrj PTAL, thanks!

File: test/e2e/scaffold/cluster_config.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (tao12345666333) [reply test/e2e/scaffold/cluster_config.go:51]: I agree with @lingsamuel

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**